Haltestellenmonitor1-DD.xcodeproj/project.pbxproj

```diff
@@ -1398,7 +1415,7 @@
 				CODE_SIGN_ENTITLEMENTS = MonitorIntents/MonitorIntents.entitlements;
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 2;
-				DEVELOPMENT_TEAM = WGL37RNHJ6;
+				DEVELOPMENT_TEAM = 9FV68KG3UM;
```

Comment:
Please do not change my development team. I will have problems on build for the app store.